contrib/examples/actions/echochain.meta.yaml

```diff
@@ -10,4 +10,11 @@ runner_type: "action-chain"
 entry_point: "chains/echochain.yaml"
 
 enabled: true
-parameters: {}
+parameters:
+  skip_notify:
```

Comment:
Still confused - what has CLI to do with it?

We have a native list type and we should use that? And if you "comma delimited string" syntax in the CLI and the type of the param is an array, CLI will automatically parse it and send it as a list so there should be no issues.